src/Exceptions/InvalidFieldsQuery.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Spatie\QueryBuilder\Exceptions;
+
+use Illuminate\Http\Response;
+use Illuminate\Support\Collection;
+
+class InvalidFieldsQuery extends InvalidQuery
```

Comment:
Can we rename this class to `InvalidFieldQuery`? That's a bit more in line with the other exceptions :)